app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+  validates :name, presence: true,
+  length: {maximum: Settings.validate.names.max_length}
```

Comment:
```suggestion
    length: {maximum: Settings.validate.names.max_length}
```
self check indent em ơi